### ISSUE TITLE
can not download pretained model 

### ISSUE THREAD

**Issue by liyingying05** [zhanghang1989/PyTorch-Encoding#90]
I can't download resnet50 and resnet101 model pretrained on the pcontext dataset,  since my machine is off-line, it might be something wrong when I download it from the url like "https://hangzh.s3.amazonaws.com/encoding/models/encnet_resnet50_pcontext-969062a5.zip", could you please provide a useful url? thanks.

**Comment by my462** [comment]
This link works for me.
"https://hangzh.s3.amazonaws.com/encoding/models/encnet_resnet50_pcontext-7846a2f0.zip"

**Comment by zhanghang1989** [comment]
`wget https://hangzh.s3.amazonaws.com/encoding/models/encnet_resnet50_pcontext-7846a2f0.zip`

**Comment by Monibsediqi** [comment]
I've the same issue. Cannot download the pretrained model (resnet50) 

Resolving hangzh.s3.amazonaws.com (hangzh.s3.amazonaws.com)... 52.219.120.114
Connecting to hangzh.s3.amazonaws.com (hangzh.s3.amazonaws.com)|52.219.120.114|:443... connected.
HTTP request sent, awaiting response... 403 Forbidden
2020-05-04 20:12:23 ERROR 403: Forbidden

**Comment by Landerku** [comment]
Yes I can't download it too from 'https://hangzh.s3.amazonaws.com/encoding/models/resnet50-25c4b509.zip', is there any problem with this site?

**Comment by zhanghang1989** [comment]
The url has been updated. Please try `pip install torch-encoding --pre`

**Comment by zhanghang1989** [comment]
The url now is `https://hangzh.s3-us-west-1.amazonaws.com/encoding/models/resnet50-25c4b509.zip`

**Comment by Landerku** [comment]
Thank you very much! 感谢！

**Comment by Landerku** [comment]
I'm so sorry that this URL [https://hangzh.s3-us-west-1.amazonaws.com/encoding/models/resnet50-25c4b509.zip](url) still cannot work, the same problem as I mentioned when downloading alerts '403 forbidden'.

**Comment by zhanghang1989** [comment]
Sorry, it is my bad. There is no such model. I think you are using an old version of this package. The supported models are https://github.com/zhanghang1989/PyTorch-Encoding/blob/master/encoding/models/model_store.py#L19

**Comment by Landerku** [comment]
Ok, I'll try this model, thank you for your patience help.

**Comment by Meoooww** [comment]
我有一个更好的方法，你可以把这个连接复制到迅雷中，就可以把这个权重文件下载下来
